support-frontend/assets/components/subscriptionCheckouts/address/postcodeFinder.jsx

```diff
@@ -59,9 +60,21 @@ const InputWithButton = ({ onClick, isLoading, ...props }) => (
   </div>
 );
 
-const ComposedInputWithButton = compose(withLabel, asControlled, withError)(InputWithButton);
-const ComposedSelect = compose(withLabel)(Select);
+const withDeliveryCheck = WappedComponent => ({ results, ...props }) => (
+  <div>
+    <WappedComponent {...props} />
+    { results.length > 0 && postcodeIsWithinDeliveryArea(props.value) && (
```

Comment:
does this run every key press, or wait until you leave the field?

Reply:
actually I need to add an onblur check

Reply:
this has been removed in favour of triggering the M25 check onSubmit